src/CoreWCF.Primitives/src/CoreWCF/Description/IPolicyExportExtension.cs

```diff
@@ -0,0 +1,10 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace CoreWCF.Description
+{
+    public interface IPolicyExportExtension
+    {
```

Comment:
I believe all your additions under Description are going to be handled by my PR as they are all WSDL related. The only change that will be needed for your work is in ServiceCredentials.